validator/sawtooth_validator/journal/block_store.py

```diff
@@ -161,13 +151,11 @@ def _build_add_block_ops(self, blkw):
         """
         out = []
         blk_id = blkw.identifier
-        with self._commit_condition:
-            out.append((blk_id, blkw.block.SerializeToString()))
-            for batch in blkw.batches:
-                out.append((batch.header_signature, blk_id))
-                for txn in batch.transactions:
-                    out.append((txn.header_signature, blk_id))
-            self._commit_condition.notify_all()
+        out.append((blk_id, blkw.block.SerializeToString()))
```

Comment:
How this works is not clear to me. It seems like the returned list is a list of tuples which might be blocks or batches or transactions. Are we not differentiating between these different types from a notification perspective?